src/dht/client.rs

```diff
@@ -503,27 +598,31 @@ impl Client {
         }
     }
 
+    /// Subscribes to a specific topic by sending a subscription command.
```

Comment:
This should mention the parameter for consistency (same for the rest of the functions in this file going downwards